src/quo/components/browser/dapp_item/component_spec.cljs

```diff
@@ -0,0 +1,12 @@
+(ns quo.components.browser.dapp-item.component-spec
+  (:require [quo.components.browser.dapp-item.view :as dapp-item]
+            [quo.foundations.resources :as quo.resources]
+            [test-helpers.component :as h]))
+(def aave (quo.resources/get-dapp :aave))
```

Comment:
can you add new lines around each of these, it's a bit cluttered and harder to read